x-pack/test/fleet_api_integration/apis/fleet_telemetry.ts

```diff
@@ -126,7 +126,11 @@ export default function (providerContext: FtrProviderContext) {
       );
     });
 
-    async function waitForAgents(expectedAgentCount: number, attempts: number, _attemptsMade = 0) {
+    async function waitForAgents(
+      expectedAgentCount: number,
+      attempts: number,
+      _attemptsMade = 0
+    ): Promise<any> {
```

Comment:
This should be `Promise<GetAgentsResponse>` that should be imported from `@kbn/fleet-plugin/common`

Reply:
Thanks for the top. I opened a PR to fix: https://github.com/elastic/kibana/pull/167488